src/layouts/applicationLayout.js

```diff
@@ -1,10 +1,24 @@
-import React from 'react'
+import React, { useState } from 'react'
 import './style.css'
 import illustration from '../assets/application/illustration.svg'
 import next from '../assets/application/next.svg'
 import { NavLink } from "react-router-dom";
+import ApplicationForm from '../layouts/forms/applicationForm';
+import DocUpload from '../layouts/forms/documentUpload';
+import FinalSubmit from '../layouts/forms/finalSubmit';
+
+const ApplicationLayout = () => {
+    const [tab,setTab] = useState(0);
+    const handleChange=()=>{
+        setTab(0);
+    }
+    const handleChange1=()=>{
+        setTab(1);
```

Comment:
instead of multiple handleChange with weird names try to use a single one and pass down tab number as argument

Reply:
sure sir. sir will you please explain a lit bit more

Reply:
Like making function 

```
const handleChange = (tabNumber) =>{
    setTab(tabNumber);
}
```

and then passing tabNumber as parameter

Reply:
Thank you sir, got it!